policyengine_us/variables/gov/states/sc/tax/income/sc_taxable_income.py

```diff
@@ -8,3 +8,6 @@ class sc_taxable_income(Variable):
     defined_for = StateCode.SC
     unit = USD
     definition_period = YEAR
+
+    adds = ["taxable_income", "sc_additions"]
```

Comment:
Can you check how the SALT deduction is treated? Generally it's excluded to avoid a circular reference.

Reply:
I checked, the SALT deduction is excluded in sc_additions.

Reply:
@hua7450 we need to create a taxable_income.py variable which excludes the SALT deduction for the taxable_income_deductions calculation if the tax unit is itemizing